src/index.js

```diff
@@ -2,66 +2,56 @@ let listenAction
 
 export default {
   bind(el, binding) {
-    const params = binding.value || {}
+    const params = binding.value || {},
+      stickyTop = params.stickyTop || 0,
+      zIndex = params.zIndex || 1000,
+      elStyle = el.style
 
-    const stickyTop = params.stickyTop || 0
-    const zIndex = params.zIndex || 1000
-
-    el.style.position = '-webkit-sticky'
-    el.style.position = 'sticky'
+    elStyle.position = '-webkit-sticky'
+    elStyle.position = 'sticky'
 
     // if the browser support css sticky（Currently Safari, Firefox and Chrome Canary）
-    if (~el.style.position.indexOf('sticky')) {
-      el.style.top = `${stickyTop}px`
-      el.style.zIndex = zIndex
+    if (~elStyle.position.indexOf('sticky')) {
+      elStyle.top = `${stickyTop}px`
+      elStyle.zIndex = zIndex
       return
     }
 
-    const elementChild = el.firstElementChild
-    elementChild.style.cssText = `left: 0; right: 0; top: ${stickyTop}px; index: ${zIndex}`
+    const elementChild = el.firstElementChild.style
```

Comment:
I know. If you need it, you can create another variable.

Since you were accessing the same element style over and over again and not doing anything else, it is better to cache it like I did.